packages/abi-coder/src/utils/transpile-abi.ts

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable no-restricted-globals */
```

Comment:
Whats with the duplicated `transpile-abi.ts` file here?

Reply:
I internalized `transpile-abi` into `abi-coder` and `abi-typegen` so that it's not exported from the `utils` package, because removing it from `utils` package exports would be a breaking change. It's pedantic and wouldn't _actually_ affect anyone in the real world. I'm okay with it remaining in `utils` as well and imported.